packages/fxa-content-server/tests/functional/404.js

```diff
@@ -1,26 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-'use strict';
-
-const { registerSuite } = intern.getInterface('object');
```

Comment:
I opted to disable the old tests https://github.com/mozilla/fxa/blob/b6bdb865c20d8aafe3fbd249f480bf4d3655b111/packages/fxa-content-server/tests/functional.js#L70. @ashrivastava-qa do you have a preference on this? I figured once everything was migrated we could delete them.

Reply:
Yes disabling is fine for now. We can delete later.